Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM ubuntu
+FROM node:8-alpine
 
-RUN apt-get update
+# Templating from registration repo
+RUN apk update && apk add bash
+RUN apk add git
+RUN npm install npm@"~5.4.0" && rm -rf /usr/local/lib/node_modules && mv node_modules /usr/local/lib
 
-RUN apt-get install -y curl
+WORKDIR /usr/src/buzzer
+COPY . /usr/src/registration
+RUN npm install
+RUN npm run build
 
-CMD ["curl", "wttr.in/atlanta"]
+RUN apk add tzdata
+ENV TZ="/usr/share/zoneinfo/America/New_York"
```

Comment:
@petschekr do you remember why you did this?

Reply:
Yep! Timezone is set to UTC by default. This sets it to EST.